packaging/flavorgen/flavors/flavors.go

```diff
@@ -88,6 +88,39 @@ func MultiNodeTemplateWithKubeVIP() []runtime.Object {
 	return append(MultiNodeTemplate, crsResources...)
 }
 
+// MultiNodeTemplateWithWindows generates templates for a hybrid cluster with Linux and Windows worker nodes
+func MultiNodeTemplateWithWindows() []runtime.Object {
+	vsphereCluster := newVSphereCluster(nil)
+	linuxMachineTemplate := newVSphereLinuxMachineTemplate()
+	controlPlane := newKubeadmControlplane(445, linuxMachineTemplate, newKubeVIPFiles())
+	linuxKubeadmJoinTemplate := newLinuxKubeadmConfigTemplate()
+	cluster := newCluster(vsphereCluster, &controlPlane)
+	linuxMachineDeployment := newLinuxMachineDeployment(cluster, linuxMachineTemplate, linuxKubeadmJoinTemplate)
+	windowsMachineTemplate := newVSphereWindowsMachineTemplate()
+	windowsKubeadmJoinTemplate := newWindowsKubeadmConfigTemplate()
+	windowsMachineDeployment := newWindowsMachineDeployment(cluster, windowsMachineTemplate, windowsKubeadmJoinTemplate)
+
+	clusterResourceSet := newClusterResourceSet(cluster)
+	crsResources := createCrsResourceObjects(&clusterResourceSet, vsphereCluster, cluster)
+
+	// removing Storage config so the cluster controller is not going not install CSI (it is installed by the clusterResourceSet)
```

Comment:
interesting, so, is that asymmetrical WRT the linux template? or is this all changing.  

i know vsphere stroage driver doesnt **yet** support windows, is that the reasoning for setting this to nil ? seems like in an ideal world you wouldnt have to do a one-off thing for the windows case.

Reply:
no with these changes, it still attempts the csi setup but through cluster resource sets. Depending on when vsphere storage driver support is expected for Windows, I can remove it from here before merge

Reply:
oh duh, we ALWAYS set this to nil, even in the MultipleNodeTempalteWKubeVip situation, so its just boilerplate i gueess